jcl/src/java.base/share/classes/openj9/internal/foreign/abi/InternalDowncallHandler.java

```diff
@@ -324,18 +346,14 @@ private final long memSegmtOfPtrToLongArg(MemorySegment argValue, LinkerOptions
 			 * to the unreachable objects.
 			 */
 			if (info != null) {
```

Comment:
There is no need for this line since each downcall will have a HeapArgInfo entry. Same for the other exception cases.